ci/recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - pyyaml
     - decorator
     # TODO: unset this pin once pandas 0.25.x is sorted out
-    - pandas 0.24.2
+    - pandas 0.25.1
```

Comment:
Thanks @David-Rod! Can you update this to _unpin_ pandas? The proposed change just pins to a different version of pandas. If there is _no pin_ specified, conda will usually grab the newest version. Thanks

Reply:
Also, please remove the `# TODO` comment above. Thanks!